### PR TITLE
google-genai[minor]: add safety settings

### PR DESCRIPTION
**Pull request by Adi8885**
Replace this entire comment with:
  - **Description:Expose safety_settings for Gemini integrations on google-generativeai
  - **Issue:NA,
  - **Dependencies:NA
  - **Twitter handle:@aditya_rane

@lkuligin for review

### CODE REVIEW

File: libs/partners/google-genai/langchain_google_genai/llms.py [hunk 1]
```diff
@@ -76,8 +80,12 @@ def _completion_with_retry(
         )
         try:
             if is_gemini:
+                print(f"kwargs : {kwargs}")
                 return llm.client.generate_content(
-                    contents=prompt, stream=stream, generation_config=generation_config
+                    contents=prompt,
+                    stream=stream,
+                    generation_config=generation_config,
+                    safety_settings=kwargs.pop("safety_settings", None),
```
Comment (lkuligin) [inline libs/partners/google-genai/langchain_google_genai/llms.py:88]: how are safety_settings passed to the kwargs?

**Review by lkuligin — CHANGES_REQUESTED** [review]

File: libs/partners/google-genai/langchain_google_genai/llms.py [hunk 2]
```diff
@@ -76,8 +80,12 @@ def _completion_with_retry(
         )
         try:
             if is_gemini:
+                print(f"kwargs : {kwargs}")
```
[the remark this answers is not in the document]
Comment (Adi8885) [reply libs/partners/google-genai/langchain_google_genai/llms.py]: Removed

File: libs/partners/google-genai/langchain_google_genai/llms.py (hunk 1 above)
Comment (Adi8885) [reply libs/partners/google-genai/langchain_google_genai/llms.py:87]: Added logic for safety setting to _generate()

Comment (lkuligin) [reply libs/partners/google-genai/langchain_google_genai/llms.py:87]: and `_stream`?

**Review by lkuligin — CHANGES_REQUESTED** [review]

File: libs/partners/google-genai/langchain_google_genai/llms.py (hunk 1 above)
Comment (Adi8885) [reply libs/partners/google-genai/langchain_google_genai/llms.py:87]: Added

**Review by lkuligin — APPROVED** [review]

**Comment by lkuligin** [comment]
@efriis please, take a look when you have time.

**Review by efriis — APPROVED** [review]

**Pull request merged**